gadget/gadget_test.go

```diff
@@ -1947,3 +1947,22 @@ volumes:
 	err = gadget.IsCompatible(gi, giNew)
 	c.Check(err, IsNil)
 }
+
+type gadgetSizeTestSuite struct{}
+
+var _ = Suite(&gadgetSizeTestSuite{})
+
+func (s *gadgetSizeTestSuite) TestIECString(c *C) {
+	for _, tc := range []struct {
+		size gadget.Size
+		exp  string
+	}{
+		{123 * gadget.SizeKiB, "123KiB"},
```

Comment:
Can you add a test for 512 bytes please?

Reply:
Fixed now